papaparse.js

```diff
@@ -42,7 +42,7 @@
 
 
 	var IS_WORKER = !global.document && !!global.postMessage,
-		IS_PAPA_WORKER = IS_WORKER && /(\?|&)papaworker(=|&|$)/.test(global.location.search),
+		IS_PAPA_WORKER = IS_WORKER && (/(\?|&)papaworker(=|&|$)/.test(global.location.search) || isBlobURL(global.location)),
```

Comment:
Indeed insted of testing if it's blob, we should probably use a diferent url. For example ending with "worker"

Reply:
Humm, you mean the blob url should end with "worker"?
I don't know if it is possible nowadays.

Reply:
I mean that we should not decide if it's a worked bassed blob, but using a diferent url. 

I think it's legitime to load paparse using a blob. With the current code it will asume allways it's a worker, which may not be true. That's why I'm sugesting to add a sufix to determine if it's worker or not.

Reply:
You are right, but I don't have any other idea since the suffix didn't work for me.

Reply:
Could you explain what you have tested, what is working and what is not working? 
I can provide a better advice if you give more details :)

Reply:
Of course,

I tried adding information to the querystring of the blob url.

Ex:
blob:https://example.org/9115d58c-bcda-ff47-86e5-083e9a215304?papaworker=true

But the url goes 404 when I do that, it only accepts the url as is.

Reply:
Well searching a little more, I can pass other information in the url, but the method is different, is via fragments: 

http://jsfiddle.net/thpf584n/1/

Reply:
Ok, I see the problem now. Lets forget about the suffix and use diretcly the Blob URL without SCRIPT_PATH_AS_BLOB_URL. If there are some problems with this design we should probably reintroduce the flag to enable blob loading in the future. 

Sorry for not giving the best advices.